Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs

```diff
@@ -128,14 +131,15 @@ private IEnumerator SetUserDecision(Patcher.UserDecision userDecision)
             Close();
         }
 
-        void OpenFile(string path)
+        void OpenFile(string path, bool isFile)
```

Comment:
If this function should open directories as well please rename it to `OpenFileSystemEntry`.

Reply:
Done